include/bluetooth/audio/capabilities.h

```diff
@@ -0,0 +1,260 @@
+/* @file
+ * @brief Internal APIs for Audio Capabilities handling
+ *
+ * Copyright (c) 2021 Intel Corporation
+ * Copyright (c) 2021-2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_CAPABILITIES_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_CAPABILITIES_H_
+
+#include <bluetooth/audio/audio.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Get list of capabilities by type */
+sys_slist_t *bt_audio_capability_get(uint8_t type);
+
+#define BT_AUDIO_CAPABILITY_UNFRAMED_SUPPORTED     0x00
+#define BT_AUDIO_CAPABILITY_UNFRAMED_NOT_SUPPORTED 0x01
```

Comment:
This would seem to be a different kind of capability type, which would speak in favour of using the full enum type instead of `uint8_t`, to make it clear exactly what value domain you're dealing with.

Reply:
I moved the values into an enum, but will not use the enum as the values are assigned to a __packed struct.